### ISSUE TITLE
Camelcase does not work with definitions / properties.

### ISSUE THREAD

**Issue by jamespearson** [manifoldco/swagger-to-ts#113]
Here is a sample from my swagger.yml ...

```
definitions:
  business_object:
    type: object
    properties:
      id:
        type: string
      type:
        type: string
      attributes:
        "$ref": "#/definitions/business_attributes"
    required:
    - id
    - type
    - attributes
  business_attributes:
    type: object
    properties:
      id:
        type: integer
      title:
        type: string
    required:
    - id
    - title
```

Converting using:
```
npx @manifoldco/swagger-to-ts swagger.yaml  --wrapper "declare namespace OpenAPI" --output schema.d.ts --camelcase
```

The output is:
```
export interface businessObject {
    id: string;
    type: string;
    attributes: business_attributes;
  }
  export interface businessAttributes {
    id: number;
    title: string;
  }
```

businessObject -> attributes is still using the none camelCase name business_attributes

Expected results:
```
export interface businessObject {
    id: string;
    type: string;
    attributes: businessAttributes;
  }
  export interface businessAttributes {
    id: number;
    title: string;
  }
```

**Comment by dangodev** [comment]
@jamespearson thanks for finding this bug, and posting a helpful example! Just merged a PR to fix this, added a test, and published `1.6.2` to npm which should resolve the issue.

**Comment by jamespearson** [comment]
Works as expected in 1.6.2 :) Thanks for the great work.
